opencti-platform/opencti-graphql/src/modules/notification/notification-domain.ts

```diff
@@ -48,22 +50,19 @@ const extractUniqRecipient = async (
   user: AuthUser,
   triggerInput: TriggerDigestAddInput | TriggerLiveAddInput,
   type: TriggerType
-): Promise<{ id: string, name: string }> => {
-  if (triggerInput.recipients && triggerInput.recipients.length > 0) {
+): Promise<BasicStoreEntity> => {
+  const { recipients } = triggerInput;
+  let recipient = user.id;
+  if (recipients?.at(0)) {
     if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES)) {
       throw ForbiddenAccess();
     }
-    if (triggerInput.recipients.length > 1) {
+    if (recipients?.at(1)) {
```

Comment:
Why this changes ?
isn't that more readable like this -> recipients?..length > 1 ?

Reply:
fixed